Makefile

```diff
@@ -126,3 +151,31 @@ vagrant-up: image.tar ## Boot the vagrant based test VM
 	# Retry in case provisioning failed because of some temporarily unavailable
 	# remote resource (like the VM image)
 	vagrant up || vagrant up || vagrant up
+
+.PHONY: check-release-version
+check-release-version:
+ifndef RELEASE_VERSION
+	$(error RELEASE_VERSION must be defined)
+endif
+
+.PHONY: commit-release-version
+commit-release-version: check-release-version
+	echo $(RELEASE_VERSION) > VERSION
+	git add VERSION
+	git commit -m "Release v$(RELEASE_VERSION)"
```

Comment:
Given that `git log` contains commits with Signed-of-by, maybe `git commit -s -m ...` ?